src/h5web/visualizations/line/hooks.ts

```diff
@@ -6,23 +6,14 @@ import { findDomain, getAxisScale } from '../shared/utils';
 import { AxisSystemContext } from '../shared/AxisSystemProvider';
 import { useVisProps } from '../../dataset-visualizer/VisProvider';
 
-export function useData(): number[] {
-  const { values, slicingIndices } = useVisProps();
-
-  if (slicingIndices) {
-    return (values as number[][])[slicingIndices[0]];
-  }
-
-  return values as number[];
-}
-
 export function useDataDomain(): Domain | undefined {
-  const data = useData();
-  return useMemo(() => findDomain(data), [data]);
+  const { values } = useVisProps();
+  return useMemo(() => findDomain(values.data as number[]), [values.data]);
```

Comment:
`values.data` is the underlying 1D JS array used by `ndarray` to store the values. I will use it everything we need the values as `number[]`

Reply:
I thin you can just specify `values` instead of `values.data` in the dependency array.